app/components/public/add-to-calender.hbs

```diff
@@ -9,15 +9,43 @@
       {{/if}}
       {{general-date @event.startsAt 'tz' tz=@event.timezone}}
     </div>
-    <UiDropdown class="ui floating dropdown button mt-4">
-      {{t 'Add to Calendar'}}
-      <div class="menu">
-        {{#each this.calendarUrls as |calendar|}}
-          <a href={{calendar.url}} class="item" target="_blank" rel="noreferrer nofollow">
-            {{calendar.name}}
-          </a>
-        {{/each}}
+
+    <div class="ui grid">
+      <div class="sixteen column row">
+        <div class="coloum">
+          <UiDropdown class="ui floating dropdown button mt-4">
+            {{#if @event.sessions}}
```

Comment:
Wrong check. See how schedule menu item is shown

Reply:
are you talking about these ?
![Screenshot from 2020-12-24 23-08-11](https://user-images.githubusercontent.com/57146965/103102104-80577a80-4640-11eb-9ab0-b45da841af30.png)

Reply:
That's call for speakers, see schedule menu item